### PR TITLE
Update user documentation for MFA enforcement

### PR DESCRIPTION
**Pull request by carolstran**
### Description

Updated user documentation to provide clear and comprehensive guidance to users on how to set up and use MFA now that we'll start enforcing it January 2024.

### Updated pages

Listed in order of change volume:

- `/how-tos/multi-factor-authentication`
- `/glossary`
- `/portal/user-settings`
- `/quickstart`
- `/how-tos`

### Additional Context

Internal 🎫 [CI-2605](https://emnify.atlassian.net/browse/CI-2605)

[CI-2605]: https://emnify.atlassian.net/browse/CI-2605?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by jeffstahlnecker — APPROVED** [review]

File: docs/how-tos/multi-factor-authentication.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+## Supported methods
+
+emnify supports the following MFA methods:
+
+- **Email-based MFA** (default method)  
+A one-time code or an authentication link is sent to the email address associated with the account.
+- **Time-based one-time password (TOTP) MFA**  
+One-time codes are generated using a [third-party authenticator app](#third-party-authenticator-apps) (for example, Google Authenticator) and entered during the login process.
+
+Once you've configured either method, you'll need to enter a one-time code to verify your identity if you log in to the Portal on a device you haven't [set as trusted](#manage-trusted-devices), use a different browser, or change your password.
```
Comment (martin-scholz-emnify) [inline docs/how-tos/multi-factor-authentication.md:34]: e-mail can't be configured so if you are enforced you are on e-mail mfa. Only if you configure TOTP you are on TOTP. 
Customers on eval can only be on TOTP as the are not forced. 
Enforcing is atm just for Enterprise customers of emnify.

File: docs/how-tos/multi-factor-authentication.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+- Google Authenticator ([Google Play Store](https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2&pli=1) or [App Store](https://apps.apple.com/us/app/google-authenticator/id388497605))
+- [Microsoft Authenticator](https://www.microsoft.com/en/security/mobile-authenticator-app)
+
+## Manage trusted devices
+
+Reaching for your device and entering a one-time password every time you need to log in to your account can be a pain, especially if you usually sign in from the same device.
+To bypass this process, select **Trust this device** before entering the MFA code from your authentication app.
+
+Then, [**Trusted Devices**](https://portal.emnify.com/user-settings#trusted-devices) should be visible in the **Two-Factor Authentication** section of **User Settings**.
+These devices remain trusted for 30 days.
```
Comment (martin-scholz-emnify) [inline docs/how-tos/multi-factor-authentication.md:131]: 90 days

**Review by martin-scholz-emnify — CHANGES_REQUESTED** [review]

File: docs/how-tos/multi-factor-authentication.md (hunk 2 above)
Comment (carolstran) [reply docs/how-tos/multi-factor-authentication.md:131]: I had a call with Lydia yesterday where she said it was going to be reduced to 30 days 🤔 but if the current system is 90 days, then I can put that in the docs now and then update again once it's reduced to 30 days. Wdyt?

Comment (carolstran) [reply docs/how-tos/multi-factor-authentication.md:131]: Confirmed that it'll be 30 days ✅

**Pull request merged**